### DISCUSSION TITLE
when installing a version of ruby that already exists, just move on instead of asking to install it again.

### DISCUSSION THREAD

**Discussion by airtonix** [rbenv/ruby-build#1863] [Build failures]
Currently, we have a local workstation provisioning shortcut in our `package.json#scripts` section that runs a sequence of setup tasks, one of which is `rbenv install`. But there are two problems:

- it's unnecessarily interactive
- if i already have 2.5.8 installed, the options are : 
  - Y, but wait 30minutes to download, compile all over again.
  - n, but suffer exit code 1 and then the rest of our provisioning doesn't run.

Neither of those are a good experience.

Instead when 2.5.8 is already installed, these should not be interactive or exit 1. 

```
$ cat .ruby-version
2.5.8

$ rbenv install 2.5.8
$ rbenv install
```

these should force a reinstall
```
$ cat .ruby-version
2.5.8

$ rbenv install --force 2.5.8
$ rbenv install --force
```

**Comment by jordan-brough** [comment]
There is a [--skip-existing](https://github.com/rbenv/ruby-build/blob/v20220610/bin/rbenv-install#L13) flag that can be used for this purpose.
